### PR TITLE
fix the bad code comment 

### PR DESCRIPTION
**Pull request by x1957**
**What this PR does / why we need it**:
fix the bad code comment in scheduler

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
None
```

### CODE REVIEW

File: plugin/pkg/scheduler/api/types.go [hunk 1]
```diff
@@ -70,7 +70,7 @@ type PriorityPolicy struct {
 	Argument *PriorityArgument
 }
 
-// Represents the arguments that the different types of predicates take
+// PredicateArgument represents the arguments that the different types of predicates take
```
Comment (bsalamat) [inline plugin/pkg/scheduler/api/types.go]: Now that you are at it, could you fix the comments as well:

PredicateArgument represents the arguments to configure predicate functions in scheduler policy configuration.

File: plugin/pkg/scheduler/api/types.go [hunk 2]
```diff
@@ -81,7 +81,7 @@ type PredicateArgument struct {
 	LabelsPresence *LabelsPresence
 }
 
-// Represents the arguments that the different types of priorities take.
+// PriorityArgument represents the arguments that the different types of priorities take.
```
Comment (bsalamat) [inline plugin/pkg/scheduler/api/types.go]: PriorityArgument represents the arguments to configure priority functions in scheduler policy configuration.

File: plugin/pkg/scheduler/api/types.go [hunk 3]
```diff
@@ -92,14 +92,14 @@ type PriorityArgument struct {
 	LabelPreference *LabelPreference
 }
 
-// Holds the parameters that are used to configure the corresponding predicate
+// ServiceAffinity holds the parameters that are used to configure the corresponding predicate
```
Comment (bsalamat) [inline plugin/pkg/scheduler/api/types.go]: ServiceAffinity holds the parameters that are used to configure the corresponding predicate in scheduler policy configuration.

File: plugin/pkg/scheduler/api/types.go [hunk 4]
```diff
@@ -92,14 +92,14 @@ type PriorityArgument struct {
...
 type ServiceAffinity struct {
 	// The list of labels that identify node "groups"
 	// All of the labels should match for the node to be considered a fit for hosting the pod
 	Labels []string
 }
 
-// Holds the parameters that are used to configure the corresponding predicate
+// LabelsPresence holds the parameters that are used to configure the corresponding predicate
```
Comment (bsalamat) [inline plugin/pkg/scheduler/api/types.go]: LabelsPresence holds the parameters that are used to configure the corresponding predicate in scheduler policy configuration.

File: plugin/pkg/scheduler/api/v1/types.go [hunk 5]
```diff
@@ -62,7 +62,7 @@ type PriorityPolicy struct {
 	Argument *PriorityArgument `json:"argument"`
 }
 
-// Represents the arguments that the different types of predicates take
+// PredicateArgument represents the arguments that the different types of predicates take
```
Comment (bsalamat) [inline plugin/pkg/scheduler/api/v1/types.go]: Please fix these comments as I suggested on `plugin/pkg/scheduler/api/types.go`.

**Review by bsalamat — COMMENTED** [review]
Thanks, @x1957!

File: plugin/pkg/scheduler/api/v1/types.go (hunk 5 above)
Comment (x1957) [reply plugin/pkg/scheduler/api/v1/types.go]: Done

**Pull request merged**